onnxruntime/contrib_ops/cpu/transformers/beam_search_parameters.cc

```diff
@@ -59,14 +59,10 @@ void BeamSearchParameters::ParseFromInputs(OpKernelContext* context) {
   ORT_ENFORCE(num_beams >= num_return_sequences,
               "num_return_sequences (", num_return_sequences, ") shall be be no more than num_beams (", num_beams, ")");
 
-  auto* temperature_tensor = context->Input<Tensor>(5);
-  temperature = temperature_tensor ? static_cast<float>(*temperature_tensor->Data<float>()) : 1;
-  ORT_ENFORCE(temperature > 0.0f, "temperature shall be greater than 0, got ", temperature);
-
-  auto* length_penalty_tensor = context->Input<Tensor>(6);
+  auto* length_penalty_tensor = context->Input<Tensor>(5);
   length_penalty = length_penalty_tensor ? static_cast<float>(*length_penalty_tensor->Data<float>()) : 1;
```

Comment:
This changes the order of previously defined values. why to remove it from the internal code when we want to readd this later.
Can we handle it in convert_beam_search.py by making it optional, what am I missing?

Reply:
@viboga, yes, it is not backward compatible. 
I believe that no one is used the operator in production yet, so it is fine to update the interface.